zeppelin-web/webpack.config.js

```diff
@@ -267,9 +267,13 @@ module.exports = function makeWebpackConfig () {
     progress: true,
     contentBase: './src',
     setup: function(app) {
-      app.use(
-        '/bower_components/',
-        require('express').static(path.join(__dirname, './bower_components/')));
+      app.use('**/bower_components/',
+        require('express').static(path.resolve(__dirname, './bower_components/')));
+      app.use('**/app/', require('express').static(path.resolve(__dirname, './src/app/')));
```

Comment:
But if we use nested context paths, it cause another problems like the screenshot below. So i believe **having correct directory structure for static files** is the solution to solve these kind of errors. However, it can be handed in other PRs. 

![image](https://cloud.githubusercontent.com/assets/4968473/22536311/2eaedb0e-e944-11e6-8813-327682b14c26.png)